tests/integration_test/integration_test.sh

```diff
@@ -119,25 +136,31 @@ function print_style {
 print_style "info" "Starting integration test"
 print_style "info" "$(date +"%Y-%m-%d %H:%M:%S")"
 
-set -e
-
 LIRA_ENVIRONMENT=${1}
 LIRA_MODE=${2}
 LIRA_VERSION=${3}
-PIPELINE_TOOLS_MODE=${4}
-PIPELINE_TOOLS_VERSION=${5}
-TENX_MODE=${6}
-TENX_VERSION=${7}
-SS2_MODE=${8}
-SS2_VERSION=${9}
-TENX_SUBSCRIPTION_ID=${10}
-SS2_SUBSCRIPTION_ID=${11:-"placeholder_ss2_subscription_id"}
-VAULT_TOKEN_PATH=${12}
-SUBMIT_WDL_DIR=${13}
-USE_CAAS=${14}
-USE_HMAC=${15}
-SUBMIT_AND_HOLD=${16}
-COLLECTION_NAME=${17:-"lira-${LIRA_ENVIRONMENT}"}
+LIRA_DIR=${4}
```

Comment:
Why is `LIRA_DIR` included as a parameter here when it gets set to `"${PWD}/lira"` later in the script? https://github.com/HumanCellAtlas/secondary-analysis/pull/557/files#diff-15c39745f8b1c92f369954524d0bd525R279

Reply:
Same question for `SECONDARY_ANALYSIS_DIR`: https://github.com/HumanCellAtlas/secondary-analysis/pull/557/files#diff-15c39745f8b1c92f369954524d0bd525R259 🤔

Reply:
LIRA_DIR can be a value entered by the tester - in that case, LIRA_DIR refers to a local repository that is already on the user's system. In this case (The case where LIRA means from github or from an image), the LIRA_DIR needs to be set, and means the directory of LIRA in the temp dir.

Reply:
Oh okay! The case for using a local directory makes sense 👍